docs/source/reducing_memory_usage.md

```diff
@@ -84,4 +84,22 @@ training_args = SFTConfig(..., packing=True, max_seq_length=512)
 
 Packing may cause batch contamination, where adjacent sequences influence one another. This can be problematic for some applications. For more details, see [#1230](https://github.com/huggingface/trl/issues/1230).
 
-</Tip>
\ No newline at end of file
+</Tip>
+
+## Liger for reducing peak memory usage
+
+[To complete]
+
+<hfoptions id="liger">
+<hfoption id="DPO">
+
+To use Liger for reducing peak memory usage, use the following code snippet:
+
+```python
+from trl import DPOConfig
+
+training_args = DPOConfig(..., use_liger_loss=True)
+```
+
+</hfoption>
+</hfoptions>
```

Comment:
@kashif I've added this section in the new guide for reducing memory usage, if you've words to fill it